src/Illuminate/Database/Query/Builder.php

```diff
@@ -2694,9 +2694,10 @@ protected function onceWithColumns($columns, $callback)
      * Insert a new record into the database.
      *
      * @param  array  $values
+     * @param  array  $defaultValues
      * @return bool
      */
-    public function insert(array $values)
+    public function insert(array $values, array $defaultValues = [])
```

Comment:
This is a breaking change to the method signature unfortunately.

Reply:
Not too sure what "method signature" mean but all I know is I tried to make sure it doesn't affect the code already in circulation.

Reply:
If anyone is overwriting the builder and `insert` method they'll get a fatal error.

Reply:
Is it? Ouchhhh didn`t even notice that until now. Thanks for the tip.